spec.md

```diff
@@ -157,6 +157,18 @@ progressively greater amount of knowledge."
 - Describe caveats.
 - If using an external API generator (like go-doc, js-doc, or so on), point to an external `API.md` file. This can be the only item in the section, if present.
 
+### Maintainer(s)
+**Status**: Required.
+
+**Requirements:**
+- Must be called `Maintainer` or `Maintainers`.
+- List maintainer(s) for a repository, along with one way of contacting them (e.g. GitHub link or email).
+
+**Suggestions:**
+- This should be a small list of people in charge of the repo. This should not be everyone with access rights, such as an entire organization, but the people who should be pinged and who are in charge of merging PRs and so on.
```

Comment:
Feels like it's missing a definition of "in charge". Currently it only mentions that it's people who are "in charge of merging PRs"